Mathlib/Geometry/Manifold/ContMDiff/Basic.lean

```diff
@@ -26,17 +26,18 @@ variable {𝕜 : Type*} [NontriviallyNormedField 𝕜]
   -- declare a smooth manifold `M` over the pair `(E, H)`.
   {E : Type*}
   [NormedAddCommGroup E] [NormedSpace 𝕜 E] {H : Type*} [TopologicalSpace H]
-  (I : ModelWithCorners 𝕜 E H) {M : Type*} [TopologicalSpace M] [ChartedSpace H M]
+  (I : ModelWithCorners 𝕜 E H) {M : Type*} [TopologicalSpace M] [CS : ChartedSpace H M]
```

Comment:
These are not needed, right? In this case, I'd remove them :-)
Sorry that the split made these changes a small pain to rebase.

Reply:
You mean the `CS : `, etc.? I think I still need them for `ContMDiffOn.comp_contMDiff` in `ContMDiff.of_comp_openEmbedding`. Let me know if that's not the case.

Reply:
I suppose not all three of `CS`, `CS'`, `CS''` are useful in this PR, but I still think we should keep all three if even one of them is used somewhere.